tests/Melodiia/Crud/Controller/GetAllTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Biig\Melodiia\Test\Crud\Controller;
+
+use Biig\Melodiia\Crud\Controller\GetAll;
+use Biig\Melodiia\Crud\CrudControllerInterface;
+use Biig\Melodiia\Crud\Persistence\DataStoreInterface;
+use Biig\Melodiia\Response\OkContent;
+use Biig\Melodiia\Test\TestFixtures\FakeMelodiiaModel;
+use Pagerfanta\Adapter\ArrayAdapter;
+use Pagerfanta\Pagerfanta;
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+use Prophecy\Prophecy\ObjectProphecy;
+use Symfony\Component\HttpFoundation\ParameterBag;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+
+class GetAllTest extends TestCase
+{
+    /** @var DataStoreInterface|ObjectProphecy */
+    private $dataStore;
+
+    /** @var AuthorizationCheckerInterface|ObjectProphecy */
+    private $authorizationChecker;
+
+    /** @var Request|ObjectProphecy */
+    private $request;
+
+    /** @var ParameterBag|ObjectProphecy */
+    private $attributes;
+
+    /** @var GetAll */
+    private $controller;
+
+    public function setUp()
+    {
+        $this->dataStore = $this->prophesize(DataStoreInterface::class);
+        $this->authorizationChecker = $this->prophesize(AuthorizationCheckerInterface::class);
+        $this->attributes = $this->prophesize(ParameterBag::class);
+        $this->attributes->get(CrudControllerInterface::MODEL_ATTRIBUTE)->willReturn(FakeMelodiiaModel::class);
+        $this->attributes->get(CrudControllerInterface::SERIALIZATION_GROUP, [])->willReturn([]);
+        $this->attributes->get(CrudControllerInterface::SECURITY_CHECK, null)->willReturn(null);
+        $this->attributes->get(CrudControllerInterface::MAX_PER_PAGE_ATTRIBUTE, 30)->willReturn(30);
+        $query = $this->prophesize(ParameterBag::class);
+        $query->getInt('page', Argument::cetera())->willReturn(1);
+        $this->request = $this->prophesize(Request::class);
+        $this->request->attributes = $this->attributes->reveal();
+        $this->request->query = $query->reveal();
+
+        $this->controller = new GetAll($this->dataStore->reveal(), $this->authorizationChecker->reveal());
+    }
+
+    public function testItIsIntanceOfMelodiiaController()
+    {
+        $this->assertInstanceOf(CrudControllerInterface::class, $this->controller);
+    }
+
+    public function testItReturnResourceFromDataStoreInsideOkContent()
```

Comment:
`testItSetResourceFromDataStoreInsideOkContent`

Reply:
I like how I say it, if you don't mind !